### PR TITLE
update api path to use account id and environment in path parameters

### PR DESCRIPTION
**Pull request by kk5190**
**Motivation**
update api path to use account id and environment in path parameters

### CODE REVIEW

File: src/PerzApiPhpClient.php [hunk 1]
```diff
@@ -144,11 +109,12 @@ public function getEntity(array $data) {
    *   Guzzle Exception.
    */
   public function getEntities(array $data) {
+    $account_id = $data['account_id'];
+    $environment = $data['environment'];
+    $uri = '/v3/accounts/' . $account_id . '/environments/' . $environment . '/contents';
```
Comment (kaynenh) [inline src/PerzApiPhpClient.php]: Are we okay with a hardcoded version number?

**Review by kaynenh — CHANGES_REQUESTED** [review]
One minor question about version number here

File: src/PerzApiPhpClient.php (hunk 1 above)
Comment (scor) [reply src/PerzApiPhpClient.php]: we could make it a constant in PHP I suppose.

File: src/PerzApiPhpClient.php [hunk 2]
```diff
@@ -81,48 +81,13 @@ public function __construct(
...
-        'environment' => $data['environment'],
-        'language' => $data['language'],
-        'view_mode' => $data['view_mode'],
-      ],
-    ];
-    return $this->request('get', '/v3/contents/' . $data['uuid'], $options);
-  }
-
   /**
    * Get entities in Personalization.
```
Comment (scor) [inline src/PerzApiPhpClient.php]: from Personalization

Comment (kk5190) [reply src/PerzApiPhpClient.php]: updated

File: src/PerzApiPhpClient.php (hunk 1 above)
Comment (kk5190) [reply src/PerzApiPhpClient.php]: updated : defined a constant for api version

File: README.md [hunk 3]
```diff
@@ -1,211 +1,3 @@
 # Acquia Perz CIS Client for PHP
 
-[![Build Status](https://travis-ci.org/acquia/content-hub-php.svg)](https://travis-ci.org/acquia/content-hub-php)
-
 A PHP Client library to consume the Perz CIS API.
-
-## Version Information
```
Comment (japerry) [inline README.md:7]: While I'm okay with this for the PR, I added a new ticket to get this readme updated to CIS api endpoint information. https://backlog.acquia.com/browse/DIT-630

**Review by japerry — APPROVED** [review]

**Pull request merged**